spec/features/user_views_index_page_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+# As a user I want to view a list of items So that I can pick items to review
+
+# Visiting "/" "/items" gets you the items#index page
+# Page should links to all items in database
+# Links should lead to item show pages
+
+feature "User views index page to see items" do
+  before(:each) do
+    FactoryGirl.create(:item)
+    FactoryGirl.create(:item,
+      item_url: "http://www.ikea.com/us/en/catalog/products/80176284/",
+      title: "HEMNES",
+      subtitle: "Coffee table, black-brown",
+      picture_url: "http://www.ikea.com/us/en/images/products/hemnes-coffee-table-brown__0104030_PE250678_S4.JPG",
+      price: "139.00"
+      )
+  end
+
+  scenario "User visits '/', gets redirected to /items" do
+    visit root_path
+
+    expect(page).to have_content "Welcome to EyeKea!"
+    expect(page).to have_content "HEMNES"
+    expect(page).to have_content "EKTORP"
+  end
+
+  scenario "User visits '/items', sees list of items & link to add more" do
+    visit items_path
+
+    expect(page).to have_content "Welcome to EyeKea!"
+    expect(page).to have_link "HEMNES"
```

Comment:
"HERMES" here is what we call a 'magic string'. while this test may work, someone eye-ing the test might not know why these links are supposed to be here! 

Instead, consider assigning the `item` in the before block to a variable (`foo`) and then calling `foo.title` to reduce the cognitive load that the next developer might experience.